comp120_StudentNum.tex

```diff
@@ -0,0 +1,51 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\usepackage{graphicx}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{Reflective Report}
+
+% Please do not change the subtitle
+\subtitle{COMP120 - Reflective Report}
+
+% Please put your student number in the author field
+\author{1507866}
+
+\begin{document}
+	
+\maketitle
+	
+\abstract{Abstract.}
+	
+\section{Introduction}
+...
+I feel that my three main weaknesses over the last semester have been my time management, problems solving skills and third one.
+	
+\section{Time Management}
+The first weakness I want to address is my time management skills. I often set time aside to do work and do not do enough work or do not set enough time aside to do the work. The issue with this is that I do not spend enough time on a piece of work which can lead to rushing at the last minute. This means the work is of poorer quality.
+ A way to improve on this weakness is to plan both the work and the time I will spend on the work. In depth planning of for example a program makes it more manageable to make, I have been planning previous programming assignments but not with enough detail. Using sprint plans more often is a technique that I could use to manage my time better and ensure a certain amount of work is done on an assignment in a week. I am aiming to try and plan programming projects for now on better and therefore aim to improve my planning in the next month.  Previously I planned the work however I did not do it in depth enough
+(Rewrite using hand out more)
+
+
+\section{Problem Solving}
+A second weakness from the last semester I feel are my problem skills, after completing the COMP110 SpaceChem assignment many of the solutions were not as good as they could be (change wording). Having better problem solving skills will also benefit my programming skills as being able to find better solutions for a problem would be helpful. 
+One way I intend to improve my programming skills is to complete more of SpaceChem, I found previously when playing it that levels further on gave me ideas to solve previous ones... Therefore in the next month I intend to finish SpaceChem and after that to try and redo levels to improve my solutions. Another action is to read the Think Like A Programmer textbook which addresses problem solving skills. I intend to finish the book within the next month as well. 
```

Comment:
There are other games which can help with your problem solving skills. For example, [InfiniFactory](http://www.zachtronics.com/infinifactory/).